DataFormats/NanoAOD/interface/FlatTable.h

```diff
@@ -41,6 +41,7 @@ namespace nanoaod {
       Float,
       Int,
       UInt8,
+      Int8,
```

Comment:
can you have this as last of the enum ?
the changes in the histogram of type() seen by the jenkin might be simply related to a shift here